assets/js/components/common/Mapbox.jsx

```diff
@@ -35,7 +35,7 @@ export default (props) => {
           alias: hotspot_alias,
           packetCount: h.packet_count,
           isFollowed: String(
-            props.followedHotspotStats.filter(function (followedHotspot) {
+            props.allOrganizationHotspots.filter(function (followedHotspot) {
```

Comment:
could you use orgHotspotMap to find if these hotspots are followed? orgHotspotsMap is just the allOrgHotspots array converted to a map